addon-test-support/scroll.js

```diff
@@ -0,0 +1,47 @@
+import getElementWithAssert from './-private/get-element-with-assert';
+import wait from 'ember-test-helpers/wait';
+
+
+/*
+  Triggers a paint (and therefore flushes any queued up scroll events).
+
+  @method triggerFlushWithPromise
+  @return {RSVP.Promise}
+  @private
+*/
+function triggerFlushWithPromise(){
+  return wait().then(() => {
```

Comment:
So I think `requestAnimationFrame` shouldn't be necessary at all here, and a standard `wait` should be enough. I'm not 100% sure on that, but the goal for us was to make sure that we captured all possible runloops, and we basically added an extra runloop that happens inside of a RAF, thus the need to schedule an extra RAF to clear the queue.

I'll see if I can play around and try to make sure that's true, there are some weird edge cases with scroll events in general.